test/compress/arrays.js

```diff
@@ -8,7 +8,7 @@ holes_and_undefined: {
     expect: {
         w=[1,,];
         x=[1,2,void 0];
-        y=[1,,2];
+        y=[1,,2,];
```

Comment:
Watching code history this should show that this was a very old test, but the test case didn't kept it's length. Yeah, it's a weird one. But I think it's correct to keep the hole here.